src/androidTest/java/nitezh/ministock/ExportUITests.java

```diff
@@ -97,10 +97,10 @@ private void clickExport() throws UiObjectNotFoundException  {
         preferencesListView.setMaxSearchSwipes(100);
         preferencesListView.scrollTextIntoView(export);
         preferencesListView.waitForExists(3000);
-        UiObject preferencesListItem =
```

Comment:
methods `setStock(int index, String symbolToAdd)  `and `removeStock(int index)` never used